docs/RefMan/REPLCommands.rst

```diff
@@ -0,0 +1,376 @@
+=============
+
+Commands
+--------
+``:! COMMAND``
+    Execute a command in the shell
+
+``:? TOPIC``, ``:help TOPIC``
+    Display a brief description of a function, type, or command. (e.g. :help
+    :help)
+
+    TOPIC can be any of:
+      * Specific REPL colon-commands (e.g. ``:help :prove``)
+      * Functions (e.g. ``:help join``)
+      * Infix operators (e.g. ``:help +``)
+      * Type constructors (e.g. ``:help Z``)
+      * Type constraints (e.g. ``:help fin``)
+      * ``:set``-able options (e.g. ``:help :set base``)
+
+``:ast EXPR``
+    Print out the pre-typechecked AST of a given term.
+
+``:b MODULE``, ``:browse MODULE``
+    Display information about loaded modules.
+
+    With no argument, ``:browse`` shows information about the names in scope.
+    With an argument ``M``, shows information about the names exported from 
+    ``M``.
+
+``:cd DIR``
+    Set the current working directory.
+
+``:check [EXPR]``
+    Use random testing to check that the argument always returns true. (If no 
+    argument, check all properties.)
+
+    By default, this will check up to 100 test cases, or until the possible
+    inputs are exhausted. This value can be changed with 
+    ``:set tests=[NEW VAL]``.
+
+``:debug_specialize EXPR``
+    Do type specialization on a closed expression.
+
+``:dumptests FILE EXPR``
+    Dump a tab-separated collection of tests for the given expression into a
+    file. The first column in each line is the expected output, and the
+    remainder are the inputs. The number of tests is determined by the ``tests``
+    option.
+
+``:e FILE``, ``:edit FILE``
+    Edit FILE or the currenly loaded module.
+
+``:eval EXPR``
+    Evaluate an expression with the reference evaluator.
+
+``:exhaust [EXPR]``
+    Use exhaustive testing to prove that the argument always returns true. (If
+    no argument, check all properties.)
+
+``:extract-coq``
+    Print out the post-typechecked AST of all currently defined terms, in a Coq-
+    parseable format.
+
+``:file-deps FILE``
+    Show information about the dependencies of a file.
+
+``:generate-foreign-header FILE``
+    Generate a C header file from foreign declarations in a Cryptol file.
+
+    Converts all foreign declarations in the given Cryptol file into C function
+    declarations, and writes them to a file with the same name but with a ``.h``
+    extension.
+
+``:l FILE``, ``:load FILE``
+    Load a module by filename.
+
+``:m [MODULE]``, ``:module [MODULE]``
+    Load a module by its name.
+
+``:module-deps MODULE``
+    Show information about the dependencies of a module.
+
+``:new-seed``
+    Randomly generate and set a new seed for the random number generator.
+
+``:prove [EXPR]``
+    Use an external solver to prove that the argument always returns true. (If
+    no argument, check all properties.)
+
+    If the expression is successfully proven, the screen will display ``Q.E.D``.
+    If not, the screen will display some counterexamples that return false.
```

Comment:
I don't think Cryptol ever displays _multiple_ counterexamples, does it? Perhaps change the phrasing here to "will display a counterexample that returns false".